project/PekkoDependency.scala

```diff
@@ -53,7 +53,7 @@ object PekkoDependency {
   }
 
   // Default version updated only when needed, https://pekko.apache.org/docs/pekko/current/project/downstream-upgrade-strategy.html
-  val minimumExpectedPekkoVersion = "0.0.0+26656-898c6970-SNAPSHOT"
+  val minimumExpectedPekkoVersion = "0.0.0+26671-d5a1b2ef-SNAPSHOT"
```

Comment:
Can we change this to `0.0.0+26669-ec5b6764-SNAPSHOT`, otherwise we get dependency conflicts and I remember you changing some other pekko module to `0.0.0+26669-ec5b6764-SNAPSHOT` when Scala 3 was merged into it.

Reply:
switched